### PR TITLE
republish models with hftransformersv2 flavor

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
republish models with hftransfor mersv2 flavor for all the models below:-

databricks-dolly-v2-12b
bert-base-cased
deepset-minilm-uncased-squad2
bert-base-uncased
microsoft-deberta-large
roberta-base-openai-detector
gpt2
microsoft-deberta-base
distilgpt2
databricks-dolly-v2-3b

### CODE REVIEW

**Comment by KonakanchiSwathi** [comment]
databricks-dolly-v2-12b 
[https://ml.azure.com/experiments/id/f868cd36-9ea7-400f-8830-5067656cb448/runs/dc71b237-96ed-4308-98e0-dc62dfdb37c6?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

bert-base-cased
[https://ml.azure.com/experiments/id/542b7761-3490-43ef-b97b-d260a6c7c107/runs/yellow_flag_n5xrvljd64?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
deepset-minilm-uncased-squad2
[https://ml.azure.com/experiments/id/d1b7edf7-e95a-4426-966e-b58c25480f56/runs/maroon_receipt_8hrz2lsyc9?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
bert-base-uncased
[https://ml.azure.com/experiments/id/542b7761-3490-43ef-b97b-d260a6c7c107/runs/crimson_beard_3q8zp599xx?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
microsoft-deberta-large
[https://ml.azure.com/experiments/id/2bb40d8e-7c40-466a-af5e-1d9b56cb18d0/runs/musing_brain_nmn9x45dh7?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
roberta-base-openai-detector
[https://ml.azure.com/experiments/id/daf7fc8a-4545-4911-99d1-a04fe5131a50/runs/sharp_milk_81m63rthmv?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
gpt2
[https://ml.azure.com/experiments/id/c6724364-c89d-4523-bdfe-b77bf36989e3/runs/bold_gold_rtv99bq5vp?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
microsoft-deberta-base
[https://ml.azure.com/experiments/id/18f11c6f-c6c6-4e6d-a473-77e725e90d97/runs/802d2849-64e0-460b-b099-a315e42eed4d?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
distilgpt2
[https://ml.azure.com/experiments/id/0f020411-bbcd-4256-adcb-b55ba303a32a/runs/tough_bean_5mkjb6xkcx?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
databricks-dolly-v2-3b
[https://ml.azure.com/experiments/id/4e2e916e-8b98-4068-9910-7f91b4d36260/runs/tough_wolf_sqrx7nzh5k?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 1]
```diff
@@ -4,17 +4,28 @@ path: ./
 properties:
   SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
   datasets: databricks/databricks-dolly-15k
-  evaluation-min-sku-spec: 16|0|64|128
-  evaluation-recommended-sku: Standard_NC24s_v3
+  evaluation-min-sku-spec: 8|0|28|56
+  evaluation-recommended-sku: Standard_DS4_v2
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:8]: @KonakanchiSwathi Could you please share the job url for evaluation that succeeded on DS4? 

Cc: @ShipraJain01

File: assets/models/system/databricks-dolly-v2-3b/description.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+Databricks-dolly-v2-3b is a large language model that can perform various tasks based on instructions. It is trained on the Databricks machine learning platform, which allows you to use it for commercial purposes. It is based on the Pythia-2.8b model from EleutherAI and fine-tuned on a dataset of instructions and responses created by Databricks employees. The dataset covers different domains such as brainstorming, classification, closed QA, generation, information extraction, open QA and summarization¹².
+
+Databricks-dolly-v2-3b is the smallest model in the Dolly v2 family, with 2.8 billion parameters. It is not the most advanced model, but it still shows impressive performance in following instructions that are not typical of the base model. It can generate text, answer questions, summarize articles, extract information, classify sentences, and more¹  .
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-3b/description.md:3]: Could you please remove superscritps on "more" and "Pythia-12b" etc?

File: assets/models/system/databricks-dolly-v2-3b/description.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Dolly v2 also has two larger and more powerful models:
+
+- Databricks-dolly-v2-12b, with 12 billion parameters and based on Pythia-12b¹²
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-3b/description.md:7]: please link databricks dolly 12b to the 12b model in the model catalog & 7b one appropriately.

File: assets/models/system/databricks-dolly-v2-3b/description.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Databricks-dolly-v2-7b, with 6.9 billion parameters and based on Pythia-6.9b³
+
+Source: Conversation with Bing, 1/9/2023
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-3b/description.md:10]: Not sure what the "Souce" correspond to here. Could you please elaborate?

**Pull request closed without merging**